django/contrib/postgres/aggregates/general.py

```diff
@@ -1,14 +1,63 @@
 from django.contrib.postgres.fields import JSONField
 from django.db.models.aggregates import Aggregate
+from django.db.models.expressions import F, OrderBy
 
 __all__ = [
     'ArrayAgg', 'BitAnd', 'BitOr', 'BoolAnd', 'BoolOr', 'JSONBAgg', 'StringAgg',
 ]
 
 
-class ArrayAgg(Aggregate):
+class OrderableAgg(Aggregate):
```

Comment:
Maybe this should be `OrderableAggMixin` and not inherit `Aggregate`? I'm thinking that other mixins could be added at some point.